Modules/Scripted/Endoscopy/Endoscopy.py

```diff
@@ -185,11 +187,12 @@ def setupFlythroughUI(self):
 
         # View angle slider
         viewAngleSlider = ctk.ctkSliderWidget()
+        # View angles are in degrees
         viewAngleSlider.decimals = 0
         viewAngleSlider.minimum = 30
         viewAngleSlider.maximum = 180
         viewAngleSlider.connect("valueChanged(double)", self.viewAngleSliderValueChanged)
-        flythroughFormLayout.addRow(_("View Angle:"), viewAngleSlider)
+        flythroughFormLayout.addRow(_("View Angle°:"), viewAngleSlider)
```

Comment:
For sake of consistency with how angle are set in the application, let's not use the symbol. Instead:
* explained it in the tooltip
* rename variable from `viewAngleSlider` to `viewAngleDegSlider` (consistent with how similar variable are name throughout the code base)

Reply:
Will do.

Reply:
```suggestion
        flythroughFormLayout.addRow(_("View Angle:"), viewAngleSlider)
```